airflow/hooks/base.py

```diff
@@ -18,14 +18,12 @@
 """Base class for all hooks"""
 import logging
 import warnings
-from typing import TYPE_CHECKING, Any, Dict, List
+from typing import Any, Dict, List
 
+from airflow.models.connection import Connection
 from airflow.typing_compat import Protocol
 from airflow.utils.log.logging_mixin import LoggingMixin
 
-if TYPE_CHECKING:
```

Comment:
You probably didn't mean to revert the changes in this file.

Reply:
Agree. See; https://github.com/apache/airflow/pull/15361

Reply:
Ah shoot, you're correct